Meridian59/Meridian59
# Update castle2d.roo

### What this PR make?

> It fixs the light bug we have in castle2d.roo

### How can i test this?

> First you need to add the new castle2d.roo in the run/localclient/resources, load the server (the blue arrow in the server window)  now go to the Royal Library of the Bourgeois Castle and cast darkness, the entire room should be dark now, with the old file some sectors aren't affect by light/darkness spells.

Original:

![castle2d before darkness](https://user-images.githubusercontent.com/90358455/180082272-1f06333b-79a5-4f4d-9737-fd79cf620bdd.PNG)

![castle2d before darkness pt2](https://user-images.githubusercontent.com/90358455/180082313-04e7726e-3d17-4cfc-87a9-1232dcee3dce.PNG)

Fixed:

![castle2d fixed darkness pt2](https://user-images.githubusercontent.com/90358455/180082348-1c767944-ea27-48fa-ac88-21319c640c6c.PNG)

![castle2d fixed darkness](https://user-images.githubusercontent.com/90358455/180082363-c0ab7e60-4a0b-4cb7-980e-274c1398fe31.PNG)
